webapp/package.json

```diff
@@ -22,6 +22,7 @@
     "@browser-logos/safari": "1.0.6",
     "@browser-logos/safari-technology-preview": "1.0.7",
     "@browser-logos/uc": "1.0.5",
+    "@browser-logos/web": "2.0.3",
```

Comment:
The PR title says `webkitgtk`, but this is `web` i.e. Epiphany?

Reply:
> but this is web i.e. Epiphany?

[Yes](https://github.com/alrra/browser-logos/tree/ed5b24d8a66365416f1e69d1979a55c9324b2367/src/web#web-fka-epiphany).

> The PR title says webkitgtk

Because `webkitgtk` is the name this project uses, `web` is the name `browser-logos` uses for that browser.

See also: https://wiki.gnome.org/Apps/Web.

Reply:
Yeah I'm aware of that, but they are not the same and in fact we run both browsers.

@clopez do we have a dedicated logo for webkitgtk now?

Reply:
> Yeah I'm aware of that, but they are not the same and in fact we run both browsers.

Ah ok. The logo this project uses is the same as the one `web` has. 🤷🏻‍♂️

Reply:
No, we don't have a dedicated logo for webkitgtk. Using the logo from Web (Epiphany) is fine for now I guess (actually we are already doing that since #1601). Also note that for the moment we don't run tests with Web/Epiphany automatically.

Reply:
Thanks!